zadania/Program.cs

```diff
@@ -101,15 +91,50 @@ void summary(List<Tuple<int,int,int,int>> links)
             summary[travel] = 1;
     }
```

Comment:
Indeksowanie dictionary takim stringiem to bardzo zły pomysł. Procesowanie tego dla stringa źle wpływa na wydajność. Wykorzystaj tu `Tuple<int, int, int, int>` lub inną tego typu strukturę